node_sdk/index.js

```diff
@@ -0,0 +1,63 @@
+const {Universal, Node, MemoryAccount, Crypto, AeSdk, CompilerHttp, hash, encode, Encoding, generateKeyPair} = require('@aeternity/aepp-sdk')
+const fetch = require('node-fetch')
+const fs = require('fs')
+
+const exampleSource = `
+contract Example =
+  entrypoint example(x : int) = x
+`
+
+const factorySource = `
+contract Example =
+  entrypoint example(x : int) = x
+main contract Factory =
+  stateful entrypoint create() =
+    Chain.create() : Example
+`
+
+const accounts = [
+  'ak_tWZrf8ehmY7CyB1JAoBmWJEeThwWnDpU4NadUdzxVSbzDgKjP',
+  'ak_2QkttUgEyPixKzqXkJ4LX7ugbRjwCDWPBT4p4M2r8brjxUxUYd',
+];
+const output = {
+  accounts
+};
+
+const devModeRequest = async (path) => {
+  const response = await fetch(`http://mdw.aeternity.localhost:3313${path}`, {
```

Comment:
Port 3013, right?

Reply:
This port is the port exposed exclusively by dev mode, which allows you to create key/micro blocks on demand

Reply:
Got `ECONNREFUSED`:

```
$ docker-compose -f docker-compose-dev.yml run node_sdk node index.js
Creating ae_mdw_node_sdk_run ... done

/app/node_modules/node-fetch/lib/index.js:1461
                        reject(new FetchError(`request to ${request.url} failed, reason: ${err.message}`, 'system', err));
                               ^
FetchError: request to http://mdw.aeternity.localhost:3313/emit_kb failed, reason: connect ECONNREFUSED 127.0.0.1:3313
    at ClientRequest.<anonymous> (/app/node_modules/node-fetch/lib/index.js:1461:11)
    at ClientRequest.emit (node:events:511:28)
    at Socket.socketErrorListener (node:_http_client:495:9)
    at Socket.emit (node:events:511:28)
    at emitErrorNT (node:internal/streams/destroy:151:8)
    at emitErrorCloseNT (node:internal/streams/destroy:116:3)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21) {
  type: 'system',
  errno: 'ECONNREFUSED',
  code: 'ECONNREFUSED'
}

Node.js v20.1.0
ERROR: 1
```

Reply:
Adding `DEV_MODE: "true"` and rebuilding the image was not enough. Are there any other steps missing?